### PR TITLE
feat: create entities

### PR DESCRIPTION
**Pull request by ishimwe07**
## Tasks completed

- [x] Create entities
- [x] Install `cds-routing-handler`
- [x] Install `@cap-js/cds-typer`
- [x] Install `typedi`
- [x] Import `reflect-metadata` globally 
- [x] Ignore files in `@cds-models` while linting

**PS : Before merging this PR, remember to squash the commits into one descriptive commit**

### CODE REVIEW

**Review by Bateyjosue — APPROVED** [review]
Nice work done :tada: :tada: 
#
## LGTM

**Review by Bateyjosue — APPROVED** [review]
## LGTM

File: db/schema.cds [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
+      status               : String;
+      trancheWeight        : Decimal;
+      Target               : Association to many Target
+                               on Target.BonusTranche = $self;
+      trancheParticipation : Association to many TrancheParticipation
+                               on trancheParticipation.bonusTranche = $self;
+      Location             : Association to Location;
+}
+
+entity TrancheParticipation {
```
Comment (pmy09) [inline db/schema.cds:17]: This is incomplete

File: package.json [hunk 2]
```diff
@@ -33,6 +37,7 @@
     "start": "cds-ts serve",
     "watch": "npx cds-ts watch --with-mocks --profile development",
     "watch:fe": "cds-ts watch --open amalisov.cuibono/index.html?sap-ui-xx-viewCache=false --livereload false",
+    "gen-types": "npx @cap-js/cds-typer \"*\" --outputDirectory @cds-models",
```
Comment (pmy09) [inline package.json:40]: please remove this

**Review by pmy09 — CHANGES_REQUESTED** [review]
Please address the issues

**Review by pmy09 — APPROVED** [review]

**Pull request merged**